test/ViewExtensionLibraryTests/LibraryResourceProviderTests.cs

```diff
@@ -305,6 +305,48 @@ public void PackagedCustomNodeSearchElementLoadedType()
             Assert.AreEqual(url.Url, item.iconUrl);
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void CustomNodePropertiesWindowValidateCategories()
+        {
+            var guid = Guid.NewGuid();
+            var name = "category test node";
+            var category = "base.level1.level2.level3";
+            var description = "A node for testing CN dialog categories drop-down";
+            var path = @"C:\temp\category_test_node.dyf";
+            var info = new CustomNodeInfo(guid, name, category, description, path);
+            var expectedQualifiedName = "dyf://base.level1.level2.level3.category test node";
+            var moq = new Mock<ICustomNodeSource>();
+            var element = new CustomNodeSearchElement(moq.Object, info);
+            var provider = new NodeItemDataProvider(new NodeSearchModel());
+            var item = provider.CreateLoadedTypeItem<LoadedTypeItem>(element);
+
```

Comment:
I could add an additional custom node with some shared categories as the original to make this test a bit more robust.  Let me know if you feel it is necessary or excessive.

Reply:
give it a shot, more documented behavior in tests is good, helps to act as a spec.